src/config/product.rosa.js

```diff
@@ -28,37 +37,259 @@ const config = {
+              ),
+            footer: ({ dataSets = [] } = {}) =>
+              translate('curiosity-graph.cardFooterMetric', {
+                date: moment
+                  .utc(dataSets?.[0]?.display?.dailyDate)
+                  .format(dateHelpers.timestampUTCTimeFormats.yearTimeShort),
+                testId: 'graphDailyTotalCard-footer'
+              })
+          },
+          {
+            header: ({ dataSets = [] } = {}) =>
+              translate('curiosity-graph.cardHeadingMetric', {
+                context: ['remainingCapacity', dataSets?.[0]?.display?.chartId],
+                testId: 'graphRemainingCapacityCard-header'
+              }),
+            body: ({ dataSets = [] } = {}) =>
+              translate(
+                'curiosity-graph.cardBodyMetric',
+                {
+                  context: [
+                    'total',
+                    dataSets?.[0]?.display?.remainingCapacityHasData && dataSets?.[0]?.display?.chartId
+                  ],
+                  testId: 'graphRemainingCapacityCard-body',
+                  total: helpers
+                    .numberDisplay(dataSets?.[0]?.display?.remainingCapacity)
+                    ?.format({
+                      average: true,
+                      mantissa: 5,
+                      trimMantissa: true,
+                      lowPrecision: false
+                    })
+                    ?.toUpperCase()
+                },
+                [
+                  <strong
+                    title={dataSets?.[0]?.display?.remainingCapacity}
+                    aria-label={dataSets?.[0]?.display?.remainingCapacity}
+                  />
+                ]
+              ),
+            footer: ({ dataSets = [] } = {}) =>
+              translate('curiosity-graph.cardFooterMetric', {
+                date: moment
+                  .utc(dataSets?.[0]?.display?.dailyDate)
+                  .format(dateHelpers.timestampUTCTimeFormats.yearTimeShort),
+                testId: 'graphRemainingCapacityCard-footer'
+              })
+          }
+        ]
+      }
+    },
+    {
+      filters: [
+        {
+          metric: RHSM_API_PATH_METRIC_TYPES.INSTANCE_HOURS,
+          fill: chartColorBlueLight.value,
+          stroke: chartColorBlueDark.value,
+          color: chartColorBlueDark.value,
+          query: {
+            [RHSM_API_QUERY_SET_TYPES.BILLING_CATEGORY]: CATEGORY_TYPES.PREPAID
+          }
+        },
+        {
+          metric: RHSM_API_PATH_METRIC_TYPES.INSTANCE_HOURS,
+          fill: chartColorGoldLight.value,
+          stroke: chartColorGoldDark.value,
+          color: chartColorGoldDark.value,
+          query: {
+            [RHSM_API_QUERY_SET_TYPES.BILLING_CATEGORY]: CATEGORY_TYPES.ON_DEMAND
+          }
+        },
+        {
+          metric: RHSM_API_PATH_METRIC_TYPES.INSTANCE_HOURS,
+          chartType: ChartTypeVariant.threshold
+        }
+      ],
+      settings: {
+        cards: [
```

Comment:
required: confirm the cards config can be shared, or not between both graph displays. it would save us a bit on configuration lines

Reply:
able to consolidate, both configs were the same